packages/design-system/src/styles/components/_Dialog.scss

```diff
@@ -43,14 +42,17 @@ $dialog-spacer: $spacer-4;
   background-size: 12px 12px;
   margin-left: auto;
   padding: 0 0 0 (12px + $spacer-1);
-
   &:active,
   &:hover {
     background-image: url('#{$image-path}/close--primary-darker.svg');
   }
-
-  &:focus {
-    background-image: url('#{$image-path}/close.svg');
+  @if $ds-include-focus-styles {
+    &:focus {
+      background-image: url('#{$image-path}/close.svg');
+    }
+  }
+  @else {
+    background-image: url('#{$image-path}/close--primary-darker.svg');
```

Comment:
Wow I didn't know we even had a separate svg just for this. Ideally this wouldn't be background-image, so that we can actually style the svg instead of including a separate image

Reply:
Yeah, it's not consistent with how we add images to the button component https://design.cms.gov/components/button/

I'll add a new issue to update the modal button to use an SVG instead of a background image

Reply:
Ah, I already filed one too: https://jira.cms.gov/browse/WNMGDS-519
Maybe they can be combined. 
I did want to add, the reason why the button examples are different is because the icons are optional, and user that want an icon will add a svg manually. In this case, the close button is by default and the user shouldn't really need to customize. From a user perspective, its slightly easier to have it implemented with a `background-image`. However, on the other side for certain icons that might need to be customized, or for performance sake (not having to include multiple svg files), it could be advantageous to use an svg element. So i think it will have to be a case by case basis, and we will probably also have to consider accessibility impact, like if we move the svg to markup, do we need to include `alt-text`?